### PR TITLE
Clarify reference to external REST API

### PR DESCRIPTION
**Pull request by gabrielklein**
In the document we reference a REST API documentation.

>Be aware that the documentation of the REST API may not be automatically installed, but once installed it is accessible through the openHAB dashboard. 

and

> "All available commands will be displayed in the REST API documentation as a response to the button "Try me out" (may require some user input for mandatory fields). "

My changes try to clarify the way we can add the REST API documentation.

### CODE REVIEW

**Comment by bgilmer77** [comment]
@gabrielklein thank you for your contribution - I have verified that the documentation exists under PaperUI as you have written.  Thank you.

File: configuration/restdocs.md [hunk 1]
```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
 ```
 
 The commands above have been copied from the REST API documentation for illustration. 
+
+## REST API documentation
+
+The REST API documentation is available on your openHAB installation as a custom user interface.
+
+You can add it by opening the PaperUI interace, clicking "Add-ons", selecting "Misc". You will see an addon called "REST Documentation" that can be easily installed.
```
Comment (ThomDietrich) [inline configuration/restdocs.md]: typo: "interface"

Also I'd suggest to rephrase:

> The "REST Documentation" can be installed as an add-on under the "Misc" category from inside Paper UI.

File: configuration/restdocs.md [hunk 2]
```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
 ```
 
 The commands above have been copied from the REST API documentation for illustration. 
+
+## REST API documentation
+
+The REST API documentation is available on your openHAB installation as a custom user interface.
```
Comment (ThomDietrich) [inline configuration/restdocs.md]: This interface combines documentation and the possibility to interact and experiment with the API.

File: configuration/restdocs.md [hunk 3]
```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
 ```
 
 The commands above have been copied from the REST API documentation for illustration. 
+
+## REST API documentation
```
Comment (ThomDietrich) [inline configuration/restdocs.md]: *D

File: configuration/restdocs.md [hunk 4]
```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
...
+
+The REST API documentation is available on your openHAB installation as a custom user interface.
+
+You can add it by opening the PaperUI interace, clicking "Add-ons", selecting "Misc". You will see an addon called "REST Documentation" that can be easily installed.
+
+On the welcome screen of openHAB, you will now see a new interface called "REST API" where you can easily see the documentation of the REST API and test it!
```
Comment (ThomDietrich) [inline configuration/restdocs.md]: "welcome screen of openHAB" -> "openHAB dashboard"

File: configuration/restdocs.md [hunk 5]
```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
...
+
 All available commands will be displayed in the REST API documentation as a response to the button "Try me out" (may require some user input for mandatory fields).  
 
+
```
Comment (ThomDietrich) [inline configuration/restdocs.md]: Remove this extra line.

**Comment by ThomDietrich** [comment]
Hello @gabrielklein, Hello @bgilmer77,

A bit too late but I left a few comments. It would be amazing if you could open a second PR! Thanks

**Comment by bgilmer77** [comment]
Agile is the mantra.  Publish, improve, publish, improve...  I approved this PR quickly to hopefully save someone the pain of tripping over the fact that the API was not available and having to troll through the board to find out that you had to install it.

@gabrielklein +1 to @ThomDietrichs request to open another PR please.

**Comment by ThomDietrich** [comment]
Yes no problem with that ;)

**Pull request merged**